subprojects/core/src/integTest/groovy/org/gradle/internal/operations/logging/LoggingBuildOperationProgressIntegTest.groovy

```diff
@@ -49,6 +49,13 @@ class LoggingBuildOperationProgressIntegTest extends AbstractIntegrationSpec {
 
     def operations = new BuildOperationsFixture(executer, testDirectoryProvider)
 
+    def setup() {
+        executer.beforeExecute {
+            // Don't let the incubating message interfere with logging
+            withPartialVfsInvalidation(false)
```

Comment:
Can we make sure somehow to remove this once the feature is out of incubation?

Reply:
Yes, since `withPartialInvalidation` should be removed completely.